_pages/about/README.md

```diff
@@ -59,3 +59,9 @@ We aim to create a thriving community of developers working together to make the
 If you or your company are interested in participating in this effort, please visit the [Contributing](/contributing/) page. We welcome all feedback and participation in the development of the machine learning platform.
 
 </div>
+
+<div class="col-xs-12" markdown="1">
+## Linaro’s Artificial Intelligence Initiative
+
+In 2018, Linaro launched the Artificial Intelligence Initiative, kick starting with Arm’s donation of Arm NN. The initiative aims to provide the best-in-class Deep Learning performance by leveraging Neural Network acceleration in IP and SoCs from the Arm ecosystem. Currently every IP vendor forks the runtime of each machine learning framework to integrate their hardware blocks and then tune for performance. This leads to a duplication of effort amongst all players, perpetual cost of re-integration for every new rebasing, and overall increased total cost of ownership. To find out more about the initiative and how to get involved, go to [https://www.linaro.org/engineering/artificial-intelligence/](https://www.linaro.org/engineering/artificial-intelligence/). 
```

Comment:
Suggest changing "kick starting with" to "kick started by", otherwise you've mixed tenses.

Reply:
Fair enough, happy with that change!